docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+#
+# There are here default credentials to create container monolith_db
+# by default we called 'monolith'
+#
+services:
+  monolith_database:
+    container_name: monolith_db
+    image: mysql/mysql-server:5.7
+    restart: on-failure
+    ports:
+    - "3307:3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MONOLITH_DATABASE_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${MONOLITH_DATABASE_NAME}
+      MYSQL_USER: ${MONOLITH_DATABASE_USER}
+      MYSQL_PASSWORD: ${MONOLITH_DATABASE_PASSWORD}
+  monolith_application:
+    container_name: monolith_app
+    image: salaboy/fmtok8s-monolith
```

Comment:
@mcruzdev this is not the right image, I would use this one `gcr.io/camunda-researchanddevelopment/fmtok8s-monolith` which is the one that is being built by the pipeline